samples/bluetooth/mesh/silvair_enocean/README.rst

```diff
@@ -69,6 +69,7 @@ The following table shows the mesh Silvair EnOcean composition data for this sam
    =============================  ===================
    Config Server                  Gen. Level Client
    Health Server                  Gen. OnOff Client
+   Generic DTT Server
```

Comment:
Just one comment... I feel we are missing a description of Generic DTT Server below where we say The models are used for the following purposes...